.github/workflows/check.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       run: |
         bundle config path vendor/bundle
         bundle install
-        gem install html-proofer
+        gem install html-proofer -v "<5.0.0"
```

Comment:
Would updating `ruby-version` to `2.7` on line 17 be an alternative to this? If so, might be preferable to continue the forward match rather than to lock down in place.

Reply:
Sadly, no. html-proofer `5.0.0` requires a minimum of Ruby `3.1` which will also require migrating from the now deprecated [actions/setup-ruby@v1](https://github.com/actions/setup-ruby) to [ruby/setup-ruby@v1](https://github.com/ruby/setup-ruby). Github [pages](https://pages.github.com/versions/) still runs on Ruby `2.7`, but with Jekyll `3.9.2` which does support Ruby `3.1` so there may be a pathway...